report/sections/data.tex

```diff
@@ -1,19 +1,29 @@
 \section{Data}
 
-\todo{TODO: Introduction to dataset, where does it come from?, number of files, structure, lm2, lm3, btn, labels, features, imbalance, etc}
+% Introduction to the dataset
+% What are facial landmarks?
+Key points on the face which allow to perform tasks focused on the face, such as emotion classification, are defined as facial landmarks. These facial landmarks (features), for instance the location of the outer left eye brow, are provided in the Bosphorus Database dataset in three different types of files. Files with an `.lm2' extension have 2D feature coordinates, `.lm3' have 3D feature coordinates, and finally the `.bnt' files have both 2D coordinates and corresponding 3D image coordinates, each labeled with the corresponding emotion it represents.
```

Comment:
- The first sentence here needs a little rewording.
- "for instance" -> "for example"
- wrap "for instance ... eye brow" in emdash?